mods/digimon/random-teams.js

```diff
@@ -865,13 +866,14 @@ class RandomDigimonTeams extends RandomTeams {
 				signatureMove: "Berserk Thinking",
 			},
 			"MetalGreymonVaccine": {
+				name: "MetalGreymon",
 				species: "metalgreymonvaccine",
```

Comment:
Is there a metalgreymonvaccine? Im not sure...

Reply:
yes there are two vaccine and virus, its just so the sprite displays correctly as its labeled metalgreymonvaccine on zarels end so any regular metalgreymon that is not the virus one is metalgreymon vaccine, the other files should be tweaked to account for this.

Reply:
This is fine